.github/workflows/playwright.yml

```diff
@@ -0,0 +1,33 @@
+name: Playwright
+on:
+    push  
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+      with:
+          fetch-depth: 0
+    - name: Use Node.js 18.x
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Restore/create node_modules cache
+      uses: actions/cache@v3
+      with:
+        path: '**/node_modules'
+        key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+    - name: Install dependencies
+      run: |
+        yarn install --frozen-lockfile
+    - name: Build dependencies
+      run: | 
+        yarn build
+    - name: Install Playwright Browsers
+      run: | 
+        npx playwright install --with-deps
+    - name: Run Playwright catalog tests
+      run: |
+        yarn run test:playwright:ci
```

Comment:
**question:** Would it possible to run this command only on apps that has recently changed?

Reply:
We're currently discussing into our team about the process because it's indeed too resource consuming.
I'm gonna put this PR as draft for now